pkg/controller.v1/mpi/mpijob_controller.go

```diff
@@ -1152,6 +1152,20 @@ func (jc *MPIJobReconciler) newLauncher(mpiJob *kubeflowv1.MPIJob, kubectlDelive
 			})
 	}
 
+	// Add default Intel MPI bootstrap variables if none is provided by the user
+	if !hasIntelMPIBootstrap(container.Env) {
```

Comment:
In this implementation, if only `I_MPI_HYDRA_BOOTSTRAP ` is set (`I_MPI_HYDRA_BOOTSTRAP_EXEC ` is unset), it will skip this section.

Can we check both `I_MPI_HYDRA_BOOTSTRAP` and `I_MPI_HYDRA_BOOTSTRAP_EXEC`?
And then, should we set env vars if either is unset?

Reply:
I added a mention of the extra two variables to the comments. I also modified the code to set either of the missing variables.

fyi, still waiting on the CLA, should get it sorted during this week.

Reply:
> I added a mention of the extra two variables to the comments. I also modified the code to set either of the missing variables.

Thank you for the updates! I left a few comments.

> fyi, still waiting on the CLA, should get it sorted during this week.

Thanks for letting me know!